docs/sql-programming-guide.md

```diff
@@ -1784,7 +1784,7 @@ working with timestamps in `pandas_udf`s to get the best performance, see
       <tr><th><b>Property Name</b></th><th><b>Default</b></th><th><b>Meaning</b></th></tr>
       <tr>
         <td><code>spark.sql.orc.impl</code></td>
-        <td><code>native</code></td>
+        <td><code>hive</code></td>
```

Comment:
We do not need this in the migration guide. Please create a new section for ORC

Reply:
Yep.

Reply:
Is there a reason the `impl` was changed back to the old implementation? this breaks `spark.read.orc`